### PR TITLE
User story 3 visitor sees all experiences assigned to a category

### PR DESCRIPTION
**Pull request by NatBee**
Wrote the feature test and worked through some of the errors. This is a many to many relationship and I need some help with setting up these relationships and displaying them in the show.

@stovermc 
@somedayrainbows 
@slague

### CODE REVIEW

**Comment by NatBee** [comment]
Also modified category factory because @slague changes weren't merged.

File: config/routes.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-
-  resources :experiences, only: [:index]
-
+  resources :categories, params: :name do
+    resources :experiences, only: [:index]
```
Comment (somedayrainbows) [inline config/routes.rb]: also need :show inside the brackets with :index

File: spec/features/visitor_sees_all_experiences_assigned_to_a_single_category_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe "User sees all experiences" do
+  scenario "when they visit the category show" do
+
+    arts = create(:category)
```
Comment (somedayrainbows) [inline spec/features/visitor_sees_all_experiences_assigned_to_a_single_category_spec.rb]: category = Category.create(name: "arts")

File: spec/features/visitor_sees_all_experiences_assigned_to_a_single_category_spec.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    art_walk = create_list(:experience, 2)
+
+    visit "/categories/arts"
+
+    expect(page).to have_content(arts.name)
```
Comment (somedayrainbows) [inline spec/features/visitor_sees_all_experiences_assigned_to_a_single_category_spec.rb]: I think Andrew mentioned using the string of what we expect here as a best practice, so in this case, maybe we should use expect(page).to have_content("Arts")

File: config/routes.rb (hunk 1 above)
Comment (stovermc) [inline config/routes.rb]: may want to change the nesting structure later. What is params: :name do ?

**Pull request merged**